### ISSUE TITLE
Modded chests support

### ISSUE THREAD

**Issue by Chubzik1** [P3pp3rF1y/SophisticatedStorage#122]
Modded chest cant be used to craft sophisticated chest. It would be nice if we could use chests from Woodworks for example

**Comment by P3pp3rF1y** [comment]
I could add specific crafting recipes similar to how quark is supported but doing it in general would require using chests which would make all other current chest to sophisticated chest recipes craft into one specific sophisticated chest (let's say oak) so universal compat can't be done here or would require player to install recipe pick mod to be able to let's say craft birch chest from quark's birch chest.
I will keep this ticket here and think about adding it - the thing is it's not only about adding but also about maintaining that compatibility going forward which takes time away from actually adding more features to the mod so I want to be sure I am only adding stuff that I want to keep supporting in the future.

**Comment by Chubzik1** [comment]
Yep, its understandable. Also i totaly forgot that i can craft chest directly from planks and redstone torches...

**Comment by Eye-sys** [comment]
To me, making any modded chest into one certain sophisticated chest seems acceptable!

**Comment by Lucky-56** [comment]
you could use tags like #sophisticatedstorage:base_chests/oak and give these tags to the different modded chests if the mod is loaded and then use the tag in the recipe
